lib/order.rb

```diff
@@ -0,0 +1,71 @@
+class Order
+
+  require_relative "./menu.rb"
+  require_relative "./menu_displayer.rb"
+
+  attr_reader :total, :cart, :payment_type, :payment
+
+  def initialize(payment_type = "card")
+    @menu = Menu.new
+    @cart = []
+    @total = 0
+    @payment_type = payment_type
+    @payment = false
+  end
+
+  def add(item, amount = 1)
+    item_number = 0
+    @cart << { (item_number += 1) => @menu.list[item - 1], :amount => amount }
+    update_total
+  end
```

Comment:
Do you have to reference the number of the item in the csv file? Consider making this more clear, I got confused with the argument name and attempted to use the name of the item (i.e pizza_1)

Reply:
yeah, defo, i guess i was imagining a slightly more imaginative restaurant and ideally a front end which controlled the ordering, but i think [_"magic numbers pizza"_](http://e.lvme.me/rkozb5.jpg) has a ring to it